lib/build.gradle.kts

```diff
@@ -86,28 +111,66 @@ dependencies {
     // This dependency is used internally, and not exposed to consumers on their own compile classpath.
     implementation("com.google.guava:guava:30.1.1-jre")
 
-    // Kotlin co-routines lib
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0")
-
-    // Nitric base clients
-    implementation("io.nitric:api:0.16.0")
+    implementation("com.google.code.gson:gson:2.10")
 
-    // Use the Kotlin test library.
-    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    // Kotlin co-routines lib
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
 
-    // Use the Kotlin JUnit integration.
-    testImplementation("org.jetbrains.kotlin:kotlin-test-junit")
+    // Kotlin serialization lib
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
 
     // This dependency is exported to consumers, that is to say found on their compile classpath.
     api("org.apache.commons:commons-math3:3.6.1")
 
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
+    // Protocol Buffers dependencies
+    implementation("io.grpc:grpc-stub:1.46.0")
+    implementation("io.grpc:grpc-kotlin-stub:1.3.0")
+    implementation("io.grpc:grpc-protobuf:1.50.2")
+    implementation("com.google.protobuf:protobuf-kotlin:3.21.8")
+    implementation("com.google.protobuf:protobuf-java:3.21.8")
+    implementation("io.grpc:grpc-netty:1.50.2")
```

Comment:
just making a note for this later, we may need to update this to a version that's compatible with graal native-image.